bb-pr

```diff
@@ -220,6 +224,8 @@ action_squash-merge() {
   local squash_merge_msg
   local json_payload
   local jq_transform='"\(.id)|\(.state)"'
+  #shellcheck disable=SC2016
+  local failure_jq_transform='.error | .message as $message | .fields | to_entries[] | "\($message)|\(.value)"'
```

Comment:
An interesting conundrum here; we could be 'explicit' and go directly for `.fields.merge_checks` but I couldn't find any documentation about what could be possible keys inside that block.

This is the error definition from swagger.json (which isn't that useful).

```
        "error": {
            "type": "object",
            "title": "Error",
            "description": "Base type for most resource objects. It defines the common `type` element that identifies an object's type. It also identifies the element as Swagger's `discriminator`.",
            "properties": {
                "type": {
                    "type": "string"
                },
                "error": {
                    "type": "object",
                    "properties": {
                        "message": {
                            "type": "string"
                        },
                        "detail": {
                            "type": "string"
                        },
                        "data": {
                            "type": "object",
                            "description": "Optional structured data that is endpoint-specific.",
                            "properties": {},
                            "additionalProperties": true
                        }
                    },
                    "required": [
                        "message"
                    ],
                    "additionalProperties": false
                }
            },
            "required": [
                "type"
            ],
            "additionalProperties": true
        },
```